maintainers/maintainer-list.nix

```diff
@@ -5224,6 +5224,12 @@
     githubId = 3036816;
     name = "Edgar Aroutiounian";
   };
+  fxttr = {
+    name = "Florian Büstgens";
+    email = "fb@fx-ttr.de";
+    github = "fxttr";
+    githubId = 16306293;
+  };
```

Comment:
@fxttr I gathered this from your commit logs, but let me know if I should update anything here!

Reply:
Everything fine, thanks :)